guides/SSO.md

```diff
@@ -0,0 +1,20 @@
+# How to configure SSO
+SSO require additionaly to configure TLS for application, in that example we will use self-signed cert, in case of use legal certificates please skip step 1.
+#### Step 1
```

Comment:
Should we name it OAuth0 example? Because for others there is no need to pass certs

Reply:
I added a command line example without a self-signed certificate and SSL configuration, so we can leave it as-is